src/App.tsx

```diff
@@ -4,32 +4,71 @@ import logo from "../public/FischLogo.svg";
 
 import styles from "./App.module.css";
 import { Card } from "./components/Card";
-import { useRef, useState } from "react";
+import { JSX, useEffect, useState } from "react";
+import { NewCard } from "./components/NewCard";
 
-function App() {
-  const [cardsAmount, setCardsAmount] = useState(1)
-  const [cards, setCards] = useState([{}]);
+interface AppProps {
+  basicCards: {
+    face: string;
+    back: string;
+  }[];
+}
+
+interface CardObject {
+  face: string;
+  back: string;
+}
+
+function App(props: AppProps) {
+  const importedCards = props.basicCards.map((card, index) => {
```

Comment:
**nitpick:** I totally don't get it why do you need it for? From where those props are coming from? You totally don't need it

Reply:
I don't yet know how are we going to import the cards, so I made it so it will take an array of objects containing only "front" and "back" fields, and transforms it into an array which also contains a "key" field.